tests/integration/evidence/keeper/infraction_test.go

```diff
@@ -96,6 +98,7 @@ func initFixture(t testing.TB) *fixture {
 		minttypes.ModuleName:           {authtypes.Minter},
 		stakingtypes.BondedPoolName:    {authtypes.Burner, authtypes.Staking},
 		stakingtypes.NotBondedPoolName: {authtypes.Burner, authtypes.Staking},
+		govtypes.ModuleName:            {authtypes.Burner, authtypes.Staking},
```

Comment:
I think we don't need to grant `Burner` and `Staking` permissions to this module address. Consider defining without permissions `{}`. If you agree, please fix this in the other files as well, since I see multiple instances of it.